package.json

```diff
@@ -37,7 +37,8 @@
     "db:dev": "run-s db:create:dev db:migrate:dev",
     "db:test": "run-s db:create:test db:migrate:test",
     "prepare": "husky install",
-    "precommit": "run-p lint build"
+    "precommit": "run-p lint build",
+		"start:client": "webpack serve --mode development --host 0.0.0.0"
```

Comment:
Not sure why GH is displaying it like this here. Looks fine in the file.

<img width="516" alt="Screen Shot 2021-08-18 at 3 37 17 pm" src="https://user-images.githubusercontent.com/37155488/129843214-1e110e33-25b0-4680-a187-ac44e7c8bc1e.png">

Reply:
lol, it's because of tabs.  NPM uses spaces.  Next time someone adds/removes a dependency, it'll reformat to spaces.